djoser/views.py

```diff
@@ -46,19 +48,21 @@ class RegistrationView(utils.SendEmailViewMixin, generics.CreateAPIView):
     """
     Use this endpoint to register new user.
     """
-    serializer_class = serializers.UserRegistrationSerializer
+    serializer_class = registration_serializer_class
     permission_classes = (
         permissions.AllowAny,
     )
     token_generator = default_token_generator
-    subject_template_name = 'activation_email_subject.txt'
-    plain_body_template_name = 'activation_email_body.txt'
+    subject_template_name = settings.get("ACTIVATION_EMAIL_SUBJECT")
+    plain_body_template_name = settings.get("ACTIVATION_EMAIL_BODY")
```

Comment:
Does anyone else think that while this is being done, it would be worth adding a ACTIVATION_EMAIL_BODY_HTML as well?